Source/x11/XGServer.m

```diff
@@ -596,48 +615,48 @@ static int byte_order(void)
 /**
  * Used by the art backend to determine the drawing mechanism.
  */
-- (void) getForScreen: (int)screen_number pixelFormat: (int *)bpp_number 
+- (void) getForScreen: (int)screen_id pixelFormat: (int *)bpp_number 
                 masks: (int *)red_mask : (int *)green_mask : (int *)blue_mask
 {
   Visual *visual;
   XImage *i;
   int bpp;
-#if 0
-  XVisualInfo template;
-  XVisualInfo *visualInfo;
-  int numMatches;
+// #if 0
+//   XVisualInfo template;
+//   XVisualInfo *visualInfo;
+//   int numMatches;
   
-  /*
-    We need a visual that we can generate pixel values for by ourselves.
-    Thus, we try to find a DirectColor or TrueColor visual. If that fails,
-    we use the default visual and hope that it's usable.
-  */
-  template.class = DirectColor;
-  visualInfo = XGetVisualInfo(dpy, VisualClassMask, &template, &numMatches);
-  if (!visualInfo)
-    {
-      template.class = TrueColor;
-      visualInfo = XGetVisualInfo(dpy, VisualClassMask, &template, &numMatches);
-    }
-  if (visualInfo)
-    {
-      visual = visualInfo->visual;
-      bpp = visualInfo->depth;
-      XFree(visualInfo);
-    }
-  else
-    {
-      visual = DefaultVisual(dpy, DefaultScreen(dpy));
-      bpp = DefaultDepth(dpy, DefaultScreen(dpy));
-    }
-#else
+//   /*
```

Comment:
Better remove this completely then to comment it out in yet another way. Now it would be commented out three times.

Reply:
You're right, done.